tests/integration/jpa/appl/pom.xml

```diff
@@ -124,6 +125,33 @@ limitations under the License.
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>${basedir}/src/main/resources</directory>
+                <excludes>
+                    <exclude>META-INF/microprofile-config.properties</exclude>
+                    <exclude>META-INF/persistence.xml</exclude>
+                </excludes>
+            </resource>
+            <resource>
+                <directory>${basedir}/src/main/resources/META-INF</directory>
+                <targetPath>META-INF</targetPath>
+                <includes>
+                    <include>microprofile-config.properties</include>
+                    <include>persistence.xml</include>
+                </includes>
+                <filtering>true</filtering>
+            </resource>
+        </resources>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>io.fabric8</groupId>
+                    <artifactId>docker-maven-plugin</artifactId>
+                    <version>0.33.0</version>
```

Comment:
Is there a specific reason for hard-coding a version here?
If not, please define this plugin management entry in the top level pom.

Reply:
Moved to helidon-applications-project